projects/app/src/data/hanzi.ts

```
@@ -0,0 +1,481 @@
+    return { type: `LeafUnknownCharacter`, strokeCount };
+  }
+
+  return { type: `LeafCharacter`, character: char as HanziChar };
+}
+
+export function strokeCountPlaceholderOrNull(
+  charOrCharPoint: string | number,
+): number | undefined {
+  const charCodePoint =
+    typeof charOrCharPoint === `string`
+      ? charOrCharPoint.codePointAt(0)
```

Biome (1.9.4)

[error] Invalid `typeof` comparison value: this expression is not a string literal

not a string literal

(lint/suspicious/useValidTypeof)